doc/manual/meson.build

```diff
@@ -0,0 +1,75 @@
+if get_option('doc')
+  ignore_headers = [
+    'private.h',
+    'pkcs11.h',
+    'conf.h',
+    'debug.h',
+    'dict.h',
+    'filter.h',
+    'log.h',
+    'mock.h',
+    'modules.h',
+    'pkcs11.h',
+    'pkcs11i.h',
+    'pkcs11x.h',
+    'private.h',
+    'client.h',
+    'proxy.h',
+    'rpc.h',
+    'rpc-message.h',
+    'util.h',
+    'virtual.h',
+    'virtual-fixed.h',
+    'array.h',
+    'compat.h'
+  ]
+
+  content_files = [
+    'p11-kit-config.xml', 'p11-kit-sharing.xml',
+    'p11-kit-devel.xml',
+    'p11-kit-proxy.xml',
+    'p11-kit-trust.xml',
+    'p11-kit-remoting.xml',
+    'p11-kit.xml',
+    'pkcs11.conf.xml',
+    'trust.xml',
+    'annotation-glossary.xml'
+  ]
+
+  version_conf = configuration_data()
+  version_conf.set('VERSION', meson.project_version())
+  configure_file(
+    input : 'version.xml.in',
```

Comment:
In other `meson.build` files there wasn't an space before `:`. It would be nice to make this consistent,.

My personal preference is to avoid spaces before the `parameter`/`value` separator.